.github/workflows/release-vsphere-template.yaml

```diff
@@ -71,3 +71,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
           VSPHERE_SERVER: ${{ secrets.VSPHERE_SERVER }}
           VSPHERE_DATASTORE: ${{ secrets.VSPHERE_DATASTORE }}
+          VSPHERE_DATACENTER: dc1
```

Comment:
```suggestion
          VSPHERE_DATACENTER: ${{ secrets.VSPHERE_DATACENTER }}
```

Reply:
Now that we have atleast 2 places to configure, wdyt about moving it into a secrets to be consistent with the other vars, and make it easier to update in the future (I added the secret in this repo already, feel free to use it or not)